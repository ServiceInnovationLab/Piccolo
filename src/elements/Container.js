import styled from 'styled-components';

const Container = styled.div`
  margin: 0 auto;
  width: 75%;
  @media (min-width: 1024px) {
    width: 88%;
  }
  @media (min-width: 1440px) {
    width: 65%;
  }
`;

export default Container;
