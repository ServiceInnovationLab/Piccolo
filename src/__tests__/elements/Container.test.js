import React from 'react';
import renderer from 'react-test-renderer';
import Container from '../../elements/Container';
import 'jest-styled-components';

test('Container renders correctly', () => {
  const tree = renderer.create(<Container />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule('width', '75%');
});
