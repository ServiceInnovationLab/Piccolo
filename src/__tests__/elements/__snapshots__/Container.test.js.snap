// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly 1`] = `
.c0 {
  margin: 0 auto;
  width: 75%;
}

@media (min-width:1440px) {
  .c0 {
    width: 55%;
  }
}

<div
  className="c0"
/>
`;
